mantidproject/mantid
# Make one percent culling in IntegrateEllipsoids and IntegrateEllipsoidsTwoStep optional

Fixes #20195 

The algorithms `IntegrateEllipsoids` and `IntegrateEllipsoidsTwoStep` sum the counts around a ellipsoidal peak and separately the counts in a ellipsoidal-shell-like volume in the background around the peak. They  then subtracts the integrated background from the integrated peak. Since the peak can spill into the background region, the top 1% of `MDBox`es in the background region are discarded. T For `WISH` this has become a problem since discarding `1%` of the background skews the SNR to values that make us believe we are dealing with a peak when we are actually dealing with background. Therefore the `1%` culling is being made optional in this PR. 

**To test:**

Ensure that the code makes sense. To see the differences you would have to test this on NoMachine. This might be not easily possible. 

**Release Notes** 

See [here]()

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
